README.md

```diff
@@ -16,11 +16,25 @@ If you find maintaining the list of contributors tedious, try using the [CLI too
-2. The listing is a table with the contributor's name, avatar (optional), a link of their choosing, and a listing of the different ways they've contributed (using the defined emoji).
-3. The emoji is linked to relevant information (if reasonable).
+### Mandatory
+Open source projects should include the following mandatory items in order to support the All Contributors specification:
 
-## Emoji key
+1. A "Contributors" section in a prominent site of the project repository documentation that includes a list of all project contributors
+	- The goal should be to use the most prominent site of the project documentation when feasible.  In many cases, this is the project README file
+	- Consider use of a CONTRIBUTORS file in the top level of the repository when the number of project contributors exceeds a level at which it is feasible to use the README file to acknowledge contributions.  In this case, a prominent link to the CONTRIBUTORS file should be included on the README page under the "Contributors" heading
+2. The listings should be formatted as a table with the following information about the project contributors:
+	- Name
+	- URL link to a site where more information can be learned about the contributor.  This URL may be determined by the contributor at the project's discretion.
+	- Indication of the Contribution Category in text or icon image format using the defined Contribution Categories and/or Contribution Categories emoji images (see below).
+	- A link to the Contribution Categories key in this repository when emoji images are used as indicators of Contribution Categories.
+3. Projects should define contributors as those who contribute to a project in any of the Contribution Categories at any level of contribution.  This specification is, by definition, inclusive of all contributions.  In cases where projects deviate from the Contribution Categories or require a specific contribution level in order to meet criteria as a project contributor, the project should provide explicit documentation of the definitions that satisfy the project contributor criteria in the CONTRIBUTING document or a similar prominent public document at the top level of the project repository.  This specification recommends against exclusion of an individual from the Contributor list based upon perceived level of contribution.  Instead, projects should use indicators of effort within a list that remains inclusive of all contributors to the project at any level of effort.  If used, these effort indicators are at the discretion of the project and are not currently specified.
+
+### Optional
+Projects should consider the following optional items as part of their commitment to the All Contributors specification:
+
+1. An optional link from the Contribution Category indicator to the category-specific contribution(s) can be included to provide additional information about the project contribution(s) as detailed in the key below
+
+## Key to the Contribution Categories
```

Comment:
If you rename this heading, you'll need to update the link `emoji key` below.

Personally, I'd prefer to keep it `Emoji Key`. Short and simple. But I'm flexible.